src/libraries/NetCoreAppLibrary.props

```diff
@@ -144,6 +144,76 @@
       System.Xml.XPath;
       System.Xml.XPath.XDocument;
     </NetCoreAppLibrary>
+    <AspNetCoreAppLibrary>
```

Comment:
Just curious: Doesn't this duplicate information available in the src/ projects❔

Reply:
This info is not available is src/projects in 5.0, its available in 6.0

Reply:
Actually this is being set in `libraries/Directory.Build.props` for all `Microsoft.Extensions*` projects:

https://github.com/dotnet/runtime/blob/f769fc29e5942d9f99883866f0d0651c17158955/src/libraries/Directory.Build.props#L9

So we don't need this you can just use that property instead.

Reply:
Did we apply that property as extensively in 5.0?  It might have been a mix of transport package + actual packages.  For example, I don't see this in System.IO.Pipelines.

Reply:
> Did we apply that property as extensively in 5.0? 

we just use that for transport package and not for other oob assemblies used by aspnetcoreapp